### DISCUSSION TITLE
longerking-blogs/2024/09/19/%E8%BF%9B%E7%A8%8B%E9%97%B4%E9%80%9A%E4%BF%A1-%E5%85%B1%E4%BA%AB%E5%86%85%E5%AD%98/

### DISCUSSION THREAD

**Discussion by giscus** [longerking/longerking-blogs#11] [General]
# longerking-blogs/2024/09/19/%E8%BF%9B%E7%A8%8B%E9%97%B4%E9%80%9A%E4%BF%A1-%E5%85%B1%E4%BA%AB%E5%86%85%E5%AD%98/

Just Work for Life,  Be Your Longer King!

https://longerking.github.io/longerking-blogs/2024/09/19/%E8%BF%9B%E7%A8%8B%E9%97%B4%E9%80%9A%E4%BF%A1-%E5%85%B1%E4%BA%AB%E5%86%85%E5%AD%98/

**Comment by longerking** [comment]
似乎看起来markdown绘制的mermaid流程图在部署构建后，并没有生效渲染出哦。

```mermaid
graph LR
	进程A <--> 共享内存 <--> 进程B
	进程C <--> 共享内存 <--> 进程...
```
